tests/bin/read_udp.c

```diff
@@ -0,0 +1,15 @@
+// Read and write a buffer of the size of a maximum sized UDP packet
+// in one go and fail, if it was fragmented.
+
+#include "libc.h"
+
+int main(void) {
+    char buf[65507];
+
+    ssize_t out = read(STDIN_FILENO, buf, sizeof(buf));
+
+    if (out != sizeof(buf))
+        return -1;
```

Comment:
Are we not usually returning `1` on error in these tests? Not sure if this will matter for consistency.

Reply:
I take this back. Looks like we return a variety of non-zero values in the other tests.